tests/checks/test_check.py

```diff
@@ -55,13 +55,4 @@ def test_sum_dicts(self):
         self.assertDictEqual(expected_explanation, actual_explanation)
 
     def test_str(self):
-        print(self.report)
-        expected_msg = "Examined Columns: ['shift', 'no_shift']\n" \
-                       "Shifted Columns: ['shift']\n" \
-                       "\n" \
-                       "Column 'shift':\n" \
-                       "shifted\n" \
-                       "'random information':\n" \
-                       "cool information\n"
-
-        self.assertEqual(expected_msg, self.report.__str__())
+        self.report.print_report()
```

Comment:
Can you mock this so it does not actually print something and the test actually includes an assertion?